### PR TITLE
Fixed issue with names on downloading images

### PR DESCRIPTION
**Pull request by conformist-mw**
I have fixed function `download_image`. Also fixed if `save_path` without trailing slash.

### CODE REVIEW

File: mimesis/utils.py [hunk 1]
```diff
@@ -114,6 +115,13 @@ def download_image(url: str = '', save_path: str = '',
 
     if url:
         image_name = url.rsplit('/')[-1]
+
+        # check if image name has two parts: name and ext to save it properly
```
Comment (lk-geimfari) [inline mimesis/utils.py]: I guess that it's better to always generate a random name for all images.

Comment (lk-geimfari) [reply mimesis/utils.py]: @conformist-mw What do you think? Can we remove comment and add random names for every downloaded image?

**Comment by lk-geimfari** [comment]
@conformist-mw So, I have only one comment and if you can fix it and update tests for it then we can merge it.

P.S Also, don't forget to add yourself to our list of contributors it's important.

**Comment by conformist-mw** [comment]
Added generating filename always and fix test.

**Comment by lk-geimfari** [comment]
@conformist-mw Any idea why AppVeyor tests failed?

**Comment by conformist-mw** [comment]
Error is: `Build cloud 'lw-host-14-e' does not support 'Visual Studio 2015' image.` and I have no idea what is mean.

**Comment by lk-geimfari** [comment]
@conformist-mw How about return not image name, but a path to the downloaded image? I think it will be more useful. What do you think?

**Comment by lk-geimfari** [comment]
@conformist-mw Don't forget about docstings: `Path to downloaded image`, instead `Image name`.

File: mimesis/utils.py [hunk 2]
```diff
@@ -106,16 +107,23 @@ def download_image(url: str = '', save_path: str = '',
     :param url: URL to image.
     :param save_path: Saving path.
     :param unverified_ctx: Create unverified context.
-    :return: Image name.
+    :return: Path to download image.
```
Comment (lk-geimfari) [inline mimesis/utils.py]: A path to the downloaded image.

**Comment by lk-geimfari** [comment]
@conformist-mw So, last comment and we can merge this PR. Thank you for your patience and interest.

File: mimesis/utils.py (hunk 2 above)
Comment (conformist-mw) [reply mimesis/utils.py]: Sorry :)

**Comment by lk-geimfari** [comment]
@conformist-mw Also seems like we don't cover all cases of images. I mean we should test images with/without a name too, to be sure that everything is okay.

**Comment by lk-geimfari** [comment]
@conformist-mw I'm sorry about that. I hope that my comments do not irritate you.

File: mimesis/utils.py [hunk 3]
```diff
@@ -106,16 +107,23 @@ def download_image(url: str = '', save_path: str = '',
...
         ssl._create_default_https_context = ssl._create_unverified_context
 
     if url:
         image_name = url.rsplit('/')[-1]
-        request.urlretrieve(url, save_path + image_name)
-        return image_name
+
+        splitted_name = image_name.rsplit('.')
+        if len(splitted_name) < 2:
+            image_name = '{}.jpg'.format(uuid4())
```
Comment (lk-geimfari) [inline mimesis/utils.py]: We should test this case

File: mimesis/utils.py [hunk 4]
```diff
@@ -106,16 +107,23 @@ def download_image(url: str = '', save_path: str = '',
...
+        else:
+            image_name = '{}.{}'.format(uuid4(), splitted_name[-1])
```
Comment (lk-geimfari) [inline mimesis/utils.py]: And this

File: mimesis/utils.py (hunk 3 above)
Comment (conformist-mw) [reply mimesis/utils.py]: Added.

File: mimesis/utils.py (hunk 4 above)
Comment (conformist-mw) [reply mimesis/utils.py]: Already tested.

**Comment by lk-geimfari** [comment]
@conformist-mw Great! Thank you!

**Pull request merged**